main/res/values/strings.xml

```diff
@@ -1362,6 +1362,9 @@
     <string name="cache_personal_note_upload_cancelled">Personal note upload cancelled</string>
     <string name="cache_description">Description</string>
     <string name="cache_description_table_note">Description contains table formatting which may need to be viewed at %s to be seen correctly.</string>
+    <string name="cache_description_rendering">Rendering Description, please wait…</string>
```

Comment:
"description" should start with a lower-case "d"

Reply:
done